boards/nxp/fmuk66-e/init/rc.board_mavlink

```diff
@@ -4,4 +4,5 @@
 #------------------------------------------------------------------------------
 
 # Start MAVLink on the USB port
+sercon
```

Comment:
What is blocking the same functionality?

Reply:
Hopefully just naming convention (GPIO_OTGFS_VBUS).

Reply:
BOARD_ADC_USB_CONNECTED()

Reply:
name is bad from the hisory